README.md

```diff
@@ -183,14 +183,19 @@ We recommend using `pipenv` to manage your python environment for SGTM. We've ch
 You can then run `pipenv shell` to enter the virtual environment from a specific shell session. Run `exit` to leave it.
 Alternatively, you don't have to 'enter' the virtual environment - you can instead run a command inside the virtual environment by prefixing the command with `pipenv run`.
 
-## Manual Testing
-Because SGTM doesn't currently support a "staging" deployment to test changes, manual testing is still recommended for changes you will be making. Here are step-by-step instructions on how to test manually/locally:
+## Testing on Staging
+SGTM's terraform configuration will also spin up a staging cluster that you can ues for manual testing. To only deploy changes to staging, run `terragrunt apply --target=module.sgtm-staging`. To direct webhook events to your staging cluster, use API gateway endpoint `/sgtm_staging` instead of `/sgtm`.
+
+You can also choose to create new staging clusters, by copying the `module "sgtm-staging"` in `main.tf`. Be sure to choose a new name for your staging cluster and update the `naming_suffix` parameter so that terraform deosn't override existing resources.
```

Comment:
```suggestion
You can also choose to create new staging clusters, by copying the `module "sgtm-staging"` in `main.tf`. Be sure to choose a new name for your staging cluster and update the `naming_suffix` parameter so that terraform doesn't override existing resources.
```